pyproject.toml

```diff
@@ -183,4 +183,21 @@ ignore = [
 ]
 
 [tool.ruff.lint.per-file-ignores]
-"geopandas/__init__.py" = ["F401"]
+"geopandas/__init__.py" = ["F401", "I"]
+
+[tool.ruff.lint.isort]
+extra-standard-library = ["packaging"]
+
+section-order = [
+  "future",
+  "standard-library",
+  "third-party",
+  "geo",
+  "first-party",
+  "local-folder",
+  "testing"
+]
+
+[tool.ruff.lint.isort.sections]
+"geo" = ["shapely", "pyproj"]
+"testing" = ["pytest", "pandas.testing", "numpy.testing", "geopandas.tests", "geopandas.testing"]
```

Comment:
I added some custom configuration for the import ordering, and the above settings are what I think matches most closely to what we generally do at the moment (we don't do it consistently, but for example in many files we somewhat keep the shapely imports in a separate section).

I think I like the above setup, but I am also fine with removing those two custom sections (and `section-order`, the above is the default but just with "geo" and "testing" added), and just go with the default settings.